lib/todo_lint/options.rb

```diff
@@ -0,0 +1,92 @@
+require "optparse"
+
+module TodoLint
+  # Handles option parsing for the command line application.
+  class Options
+    DEFAULT_REPORTER = ["Default", :stdout]
+
+    # Parses command line options into an options hash
+    # @api public
+    # @example Options.new.parse("todo_lint -c app.rb")
+    # @param args [Array<String>] arguments passed via the command line
+    # @return [Hash] parsed options
+    def parse(args)
+      @options = {}
+
+      OptionParser.new do |parser|
+        parser.banner = "Usage: todo_lint [options] [files]"
+
+        add_file_options parser
+        add_extension_options parser
+      end.parse!(args)
+
+      # Any remaining arguments are assumed to be files
+      @options[:files] = args
+
+      @options
+    end
+
+    private
+
+    # Adds the file options to the @options hash
+    # @api private
+    # @return [Hash]
+    def add_file_options(parser)
+      parser.on("-c", "--config config-file", String,
+                "Specify which config file you want to use") do |conf_file|
+        @options[:config_file] = conf_file
+      end
+      exclude_file_options(parser)
```

Comment:
I can imagine each method being responsible for one configuration option, and that's a kind of nice way to organize it, but I feel like this line belongs above with the others like it